examples/monique/calibration/seasonality_calibration/02_seasonality_calibration/run_calibration.py

```diff
@@ -182,5 +198,5 @@ def run_calibration(directory: str = '.', show_warnings: bool = True, **kwargs):
     # print("...done.")
 
     # Specify local folder for calibration results
-    directory = r'C:\Projects\emodpy-snt\data\TEST_DEST_CALIBRA4'
-    run_calibration(directory=directory, show_warnings=False)
+    directory = r'C:\Projects\emodpy-snt\data\TEST_DEST_CALIBRA5'

```

Comment:
should directory for calibration results be in the manifest?

Reply:
Good suggestion! Now added directory to manifest.py